src/main/java/mekanism/common/tile/TileEntityGlowPanel.java

```diff
@@ -37,6 +37,11 @@ public void setOrientation(EnumFacing newSide)
 	@Override
 	public void handlePacketData(ByteBuf dataStream) throws Exception 
 	{
+		if(Mekanism.hooks.MCMPLoaded)
+		{
+			dataStream.readByte();
```

Comment:
These are the single byte ordinal headers mentioned in the commit description. `TileEntityGlowPanel` is the only possible multipart that attaches to a face so it adds its `EnumFacing` ordinal as the header while the transmitters all use `6` (one higher than the highest possible `EnumFacing` ordinal) to indicate the center slot.

These are used by the new `MultipartTileNetworkJoiner`.